er_wheels/src/er_wheel_odometry_node.cpp

```diff
@@ -0,0 +1,147 @@
+#include "er_wheel_odometry_node.h"
+
+
+WheelOdometryNode::WheelOdometryNode() : ticks_per_rev_(3591.84/4), wheel_radius_(0.049), base_radius_(0.085), yaw_(0) {
+  angular_velocity_[MotorSide::Left] = 0;
+  angular_velocity_[MotorSide::Right] = 0;
+  // note messages are initialized with 0, so intiially the /base_frame will align with the /map frame
+}
+
+WheelOdometryNode::~WheelOdometryNode() {
+
+}
+
+void WheelOdometryNode::run_node() {
```

Comment:
as mentioned earlier, could be good to break it up a little

Reply:
Addressed in the latest commit.